src/exports.h

```diff
@@ -11,6 +11,10 @@
 #define TJFLAG_FASTDCT 0
 #endif
 
+#ifdef DEFAULT_QUALITY
+#undef DEFAULT_QUALITY
```

Comment:
Maybe `NJT_DEFAULT_QUALITY ` instead of just `DEFAULT_QUALITY`. Same for subsampling.